init.sql

```diff
@@ -7,23 +7,56 @@ CREATE ROLE readonly;
 GRANT USAGE ON SCHEMA public TO readonly;
 GRANT SELECT ON ALL TABLES IN SCHEMA public TO readonly;
 
-CREATE USER mainnet with password 'password';
-GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO mainnet;
-GRANT ALL PRIVILEGES ON ALL SEQUENCES IN SCHEMA public TO mainnet;
-
-CREATE TABLE daily_statistics
-(
-    computed_for_timestamp     numeric(20, 0) NOT NULL,
-    transactions_count         numeric(20, 0) NOT NULL,
-    teragas_used               numeric(20, 0) NOT NULL,
-    deposit_amount             numeric(45, 0) NOT NULL,
-    new_accounts_count         numeric(20, 0) NOT NULL,
-    deleted_accounts_count     numeric(20, 0) NOT NULL,
-    active_accounts_count      numeric(20, 0) NOT NULL,
-    new_contracts_count        numeric(20, 0) NOT NULL,
-    new_unique_contracts_count numeric(20, 0) NOT NULL,
-    active_contracts_count     numeric(20, 0) NOT NULL
-);
-
-ALTER TABLE ONLY daily_statistics
-    ADD CONSTRAINT daily_statistics_pkey PRIMARY KEY (computed_for_timestamp);
+CREATE TABLE daily_transactions_count
+(
+    daily_timestamp    numeric(20, 0) PRIMARY KEY,
```

Comment:
1. Do you want to use block timestamp or some time of the day timestamp? I wonder if we may just use the native timestamp type here and document that we always compute it as of the latest block of that day.
2. It might be useful for sanity checks to store block hash as of the last block we included in the computations (maybe it is even useful to store the first block of the range). WDYT? I don't see the usage outside of just sanity checks, so I am not going to insist

Reply:
I meant day timestamp at first. I thought it's better to use the same data types in both DBs, it allows us not to make any casts if we decide to merge the data somehow.
I changed the type to DB `DATE` type, it suits the best here, and it's easy to cast it to timestamp if we need it. We do not care about time, so the date should be enough. 
For each day, I include all transactions [00:00 UTC this day, 00:00 UTC next day). I think it should be intuitive, especially for daily statistics.
We do not rely on blocks here, so I prefer not to include block hashes. But I also don't see anything bad in it, so if you see the scenarios when it could be useful – we can add them.